tests/parsers/test_sql_directory_parser.py

```diff
@@ -109,3 +109,22 @@ def test_parse_creates_xcom(self):
             rendered_tasks = aql.render(dir_path + "/single_task_dag")
 
         test_utils.run_dag(self.dag)
+
+    def test_parse_to_dataframe(self):
```

Comment:
Is the goal for this test to cover the original issue (a `task` defined within a `task_group` should still be loaded into a data frame)?

If the internal implementation of `render` stops using task group, we would no longer be covering the original problem. Would it be worth to have an explicit test using task groups without `render`?